### PR TITLE
Nav Unification: Remove Hover Effect at Top of the Sidebar

### PR DESCRIPTION
**Pull request by Aurorum**
Fixes Automattic/wp-calypso#51835

#### Changes proposed in this Pull Request:

* Removes the hover effect introduced here: https://core.trac.wordpress.org/changeset/49961

#### Jetpack product discussion

* Some discussion in Automattic/wp-calypso#51835

#### Does this pull request change what data or activity we track or use?

* Nope

#### Testing instructions:

* Hover over the "Browse Sites" and site card in the Sidebar 
* Verify that no border appears

| Before                                                                                                                                                                | After                                                                                                                                                                 |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Screenshot 2021-04-12 at 19 47 37 | Screenshot 2021-04-12 at 19 47 28 |

### CODE REVIEW

File: projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.css [hunk 1]
```diff
@@ -256,6 +256,12 @@
 	}
 }
 
+/* Prevent box-shadow at the top of the sidebar */
+#adminmenu li.menu-top:hover,
+#adminmenu a.menu-top:hover {
+	box-shadow: none;
+}
+
```
Comment (cpapazoglou) [inline projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.css:264]: I am afraid that we will need to be more specific cause this removes the hover effect from all top level menu items

**Review by cpapazoglou — CHANGES_REQUESTED** [review]
👋  @Aurorum! Can you revise the CSS rule so that it targets only the "Browse Sites" and "Site Card"?
`site-switcher` and `toplevel_page_site-card` seem like good candidates.

File: projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.css (hunk 1 above)
Comment (Aurorum) [reply projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.css]: Good catch - thank you! Fixed! :)

**Review by cpapazoglou — DISMISSED** [review]
Thanks @Aurorum! LGTM!

**Comment by cpapazoglou** [comment]
WPCOM Counterpart D60146-code

**Comment by cpapazoglou** [comment]
@Aurorum can you please merge master to this branch?

**Comment by Aurorum** [comment]
Sure thing - should be done now!

**Review by mmtr — APPROVED** [review]

**Comment by cpapazoglou** [comment]
Deployed r224316-wpcom.

**Pull request merged**